### DISCUSSION TITLE
Deno

### DISCUSSION THREAD

**Discussion by jonatasdiscover** [gulpjs/gulp#2463] [Ideas] [answered]
Do you have any plans to create a [Deno](http://deno.land) version of Gulp ?

**Comment by yocontra** [comment, marked as answer]
Maybe! Once the APIs stabilize and it gains more traction - not something we have thought a lot about yet. IIRC one of deno's things it that they eliminated streams and use iterables, so anything that would be a "gulp for deno" would ultimately be a much different API.
